src/NServiceBus.Core/Routing/Legacy/ConfigureMSMQDistributor.cs

```diff
@@ -20,6 +21,7 @@ public static void EnlistWithLegacyMSMQDistributor(this EndpointConfiguration co
             config.GetSettings().Set("LegacyDistributor.Address", masterNodeAddress);
             config.GetSettings().Set("LegacyDistributor.ControlAddress", masterNodeControlAddress);
             config.GetSettings().Set("LegacyDistributor.Capacity", capacity);
+            config.DisableFeature<TimeoutManager>();
```

Comment:
yeah you're right. Actually we wouldn't even have to disable the TM as it disabled itself anyway because of the "alternative" timeout manager as far as I can see.